config/routes.rb

```diff
@@ -67,7 +67,7 @@
     end
 
     # TODO: Extract year constraint to future proof for future cycles
-    resources :recruitment_cycles, param: :year, constraints: { year: /2020|2021/ }, path: "", only: :show do
+    resources :recruitment_cycles, param: :year, constraints: { year: /2021|2022/ }, path: "", only: :show do
```

Comment:
I wonder if `Settings` is loaded by this time ... it outta be. If so, we could use that in the constraint here.

Reply:
Carded as https://trello.com/c/jwC8kKnb/373-todo-extract-year-constraint-to-future-proof-for-future-cycles

We can wonder then... it should be fine... but its out of scope for this PR as it is big enough.